app/views/guide_typography.html

```diff
@@ -214,6 +215,23 @@ <h3 class="heading-medium" id="typography-hidden-text">Hidden text (progressive
   </div>
 
 
+  <h3 class="heading-medium" id="typography-class">Use styling automatically</h3>
```

Comment:
Having the word 'scoped' in there is good. But I'm not sure if the whole expression makes it clear to people what it means, especially what the consequences and benefits are.

Thinking from a different angle, maybe this would fit?
"How to not use classes on every element"